Update the operator version check logic when generating models for testing upgraders

cccclai opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #71938
* __->__ #71894

The model generation script will check the model version, to ensure the developer run the script before they change operator

Previously, the version use the old model version. However, it's hard for developer to know the old version number. In this change, it use the current max operator version to check. It's less strict, but more developer friendly



Differential Revision: [D33804737](https://our.internmc.facebook.com/intern/diff/D33804737/)

Differential Revision: [D33804737](https://our.internmc.facebook.com/intern/diff/D33804737)

Commit:
Update the operator version check logic when generating models for testing upgraders



Differential Revision: [D33804737](https://our.internmc.facebook.com/intern/diff/D33804737/)

[ghstack-poisoned]

tugsbayasgalan commented on test/jit/fixtures_srcs/generate_models.py:
```diff
@@ -172,12 +172,11 @@ def generate_models(model_directory_path: Path):
             logger.error(
                 f"Actual model version {actual_model_version} "
-                f"doesn't match the expect model version {expect_model_version}. "
+                f"is smaller or larger than {expect_model_version}. "
```
This is not declared anywhere?

Commit:
Update on "Update the operator version check logic when generating models for testing upgraders"


Differential Revision: [D33804737](https://our.internmc.facebook.com/intern/diff/D33804737/)

[ghstack-poisoned]

Commit:
Update on "Update the operator version check logic when generating models for testing upgraders"


The model generation script will check the model version, to ensure the developer run the script before they change operator

Previously, the version use the old model version. However, it's hard for developer to know the old version number. In this change, it use the current max operator version to check. It's less strict, but more developer friendly



Differential Revision: [D33804737](https://our.internmc.facebook.com/intern/diff/D33804737/)

Differential Revision: [D33804737](https://our.internmc.facebook.com/intern/diff/D33804737)

[ghstack-poisoned]
